documentation/onboarding.md

```diff
@@ -0,0 +1,152 @@
+# Onboarding guide
+
+The Agenda application connects to external scheduling applications like Microsoft Outlook Calendar, Google Calendar, or Zoom, and merges their independent meeting data into one place. By consolidating team member availability into one simplified workspace, you can quickly schedule meetings that work well for everyone. 
+
+This document guides you through the installation process on your device, creating a new account, and linking an external calendar.
+
+## System requirements
+
+The sections below outline the system requirements for Agenda depending on whether you are using [Windows] or [MacOS].
+
+### Windows system requirements
```

Comment:
Since this and MacOS are both already nested under "System requirements", their headings inherit that context and can be shortened: "Windows"/"MacOS"